testplan/report/testing/base.py

```diff
@@ -460,10 +467,16 @@ def status(self):
 
     def merge(self, report, strict=True):
         """
-          TestCaseReport merge overwrites everything in place,
-          as assertions of a test case won't be split among different runners.
+        TestCaseReport merge overwrites everything in place, as assertions
+        of a test case won't be split among different runners. For some special
+        test cases, choose the one whose status is of higher precedence.
         """
         self._check_report(report)
+        if not self.replace_instead_of_merge and \
```

Comment:
Personally I find `\` very ugly - can be avoided with parentheses:

```suggestion
        if (not self.replace_instead_of_merge) and (
                Status.precedent([self.status]) <
                Status.precedent([report.status])):
```

as recommended by PEP 8 https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
Yes, it's better use parentheses, but the existed code already use a lot of '\' at the end of long lines. So there might be a task for me to add code style check, or I will make a new PR to change all that.